cppsdk/source/playfab/PlayFabEventApi.cpp

```diff
@@ -0,0 +1,25 @@
+#include <stdafx.h>
+
+#ifndef DISABLE_PLAYFABENTITY_API
+
+#include <playfab/PlayFabEventApi.h>
+
+namespace PlayFab
+{
+    PlayFabEventAPI::PlayFabEventAPI() :
+        eventRouter(std::make_shared<PlayFabEventRouter>()) // default event router
+    {
+    }
+
+    void PlayFabEventAPI::EmitEvent(std::unique_ptr<const IPlayFabEvent> event, PlayFabEmitEventCallback callback) const
```

Comment:
const ?

Reply:
Good catch!

---
In reply to: [227967676](https://github.com/PlayFab/XPlatCppSdk/pull/6#discussion_r227967676) [](ancestors = 227967676)